lib/vmpooler/pool_manager.rb

```diff
@@ -369,7 +369,7 @@ def _destroy_vm(vm, pool, provider)
 
         $redis.srem('vmpooler__completed__' + pool, vm)
 
-        finish = format('%.2f', Time.now - start)
+        finish = format('%<time>.2f', time: Time.now - start)
```

Comment:
Presumably this was to resolve a failure. Does the addition of 'time:' change the behavior at all?

Reply:
Yep, the [StyleFormatStringToken](https://rubocop.readthedocs.io/en/latest/cops_style/#styleformatstringtoken) cop prefers named parameters and it shouldn't change the behavior at all. I can verify that the output looks the same now.

Reply:
If you prefer not having named parameters in the format strings, we could also do `EnforcedStyle: unannotated`

Reply:
I'm fine with it, I just wanted to make sure that I followed.